### DISCUSSION TITLE
[@types/mapbox-gl]: Missing options for stretchable images in `addImage`

### DISCUSSION THREAD

**Discussion by YaroslavKormushyn** [DefinitelyTyped/DefinitelyTyped#63210] [Issues with a @types package]
Hello 👋
It seems like the `addImage` on the `Map` class is missing some options for stretchable images.
The declaration as of `2.7.7` is:
```typescript
addImage(
            name: string,
            image:
                | HTMLImageElement
                | ArrayBufferView
                | { width: number; height: number; data: Uint8Array | Uint8ClampedArray }
                | ImageData
                | ImageBitmap,
            options?: { pixelRatio?: number | undefined; sdf?: boolean | undefined },
        ): void;
```
The options should also have `stretchX`, `stretchY`, and `content`. [Source](https://docs.mapbox.com/mapbox-gl-js/example/add-image-stretchable/)

[a comment by an automated account is not shown]

**Comment by mschilde** [comment]
Thanks for the issue! Do you plan on submitting a PR for this or shall I take care?

**Reply by YaroslavKormushyn** [reply]
I'd appreciate it if you could take care of it since I have very little experience with this codebase.
